README.md

```diff
@@ -315,7 +315,7 @@ resp = s3.list_buckets
 #### [fog](http://fog.io/storage/)
```

Comment:
This link does not use `require fog/aws`. Perhaps the README should stay as is, to reflect the examples on that page?